server/public/model/channel.go

```diff
@@ -201,47 +200,6 @@ func WithID(ID string) ChannelOption {
 	}
 }
 
-// The following are some GraphQL methods necessary to return the
-// data in float64 type. The spec doesn't support 64 bit integers,
-// so we have to pass the data in float64. The _ at the end is
-// a hack to keep the attribute name same in GraphQL schema.
-
-func (o *Channel) CreateAt_() float64 {
```

Comment:
Are we okay making these changes to the public package outside of a major version? If not, we could always mark them `// Deprecated` and then remove them later

Reply:
Yeah good point. But technically, the public module is still in 0.x. So we can do what we want ;)